test/infrastructure/docker/api/v1beta1/dockerclustertemplate_types.go

```diff
@@ -27,6 +27,7 @@ type DockerClusterTemplateSpec struct {
 
 // +kubebuilder:object:root=true
 // +kubebuilder:storageversion
+// +kubebuilder:printcolumn:name="Cluster",type="string",JSONPath=".metadata.labels['machine\\.cluster\\.x-k8s\\.io/cluster-name']",description="Cluster"
```

Comment:
```suggestion
```
We don't set a cluster-name label on this resource

Reply:
Agreed, this is a template introduced for ClusterClasses, so it is shared across many clusters.